queries/buildings.jinja2

```diff
@@ -1,13 +1,20 @@
 SELECT
-    name,
+    -- drop name when the item is a POI - in that case the name
+    -- probably refers to the POI. instead, we can use the house
+    -- name tag, if it has been set. likewise for landuse areas
+    -- where we might want to label the landuse rather than the
+    -- building.
+    CASE WHEN mz_poi_min_zoom IS NULL AND
+              (mz_is_landuse IS NULL OR mz_is_landuse = FALSE)
+         THEN name
+         ELSE "addr:housename"
+    END AS name,
     way_area::bigint AS area,
     osm_id AS __id__,
-    "building:part",
+    'openstreetmap.org' AS source,
```

Comment:
I don't know. I _thought_ I was adding it for compatibility with the other layers, but it seems that we don't include `source` for the boundaries, earth, POIs or transit layers either. Transit and POIs are currently OSM-only, but earth and boundaries do mix data sources...

@nvkelso Should we be annotating the source for _all_ features, or just features in layers which aren't single-source?